infra/secrets.yaml

```diff
@@ -0,0 +1,29 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: ''
+Resources:
+  password:
+    Type: 'AWS::SecretsManager::Secret'
+    Properties:
+      Name: databasepassword
+      Description: "This is the password for the database, shared with the container."
+      GenerateSecretString:
```

Comment:
**Question** (no change required): Does this still get re-generated every time we update the stack? Just want to make sure I understand it correctly, and maybe it is a good idea to document it in `deploy-prod` script or `README`.

Reply:
Nope!  That's what we're fixing with this change - it only gets regenerated if the resource is destroyed, so on a full delete/recreate, which should happen very rarely (and would require that the database be destroyed, also).

Reply:
I see. That's great. Thanks!